kms/api-client/functions_test.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import subprocess
+
+from googleapiclient import discovery
+
+import functions
+
+
+# Your Google Cloud Platform Key Location
```

Comment:
The audience for these comments is us, the maintainers. So maybe it makes sense to either remove or revise them?